### PR TITLE
Remove prism warning when no language is provided

### PR DESCRIPTION
**Pull request by matthewp**
## Changes

- Removes the prism warning seen in the starter template.

## Testing

No

## Docs

No

### CODE REVIEW

File: packages/markdown/remark/src/remark-prism.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ function runHighlighter(lang: string, code: string) {
   let classLanguage = `language-${lang}`
 
   if (lang == null) {
-    console.warn('Prism.astro: No language provided.');
+    lang = 'shell';
```
[the remark this answers is not in the document]
Comment (natemoo-re) [reply packages/markdown/remark/src/remark-prism.ts:15]: I think we settled on `plaintext`, which disables highlighting? That seems to be the default on GitHub.

[the remark this answers is not in the document]
Comment (natemoo-re) [reply packages/markdown/remark/src/remark-prism.ts:15]: In which case, we should just skip any work done to highlight the code below this, right?

[the remark this answers is not in the document]
Comment (matthewp) [reply packages/markdown/remark/src/remark-prism.ts]: Looks like the plaintext language does exist, so no need to skip anything.

**Review by natemoo-re — APPROVED** [review]
LGTM!

**Pull request merged**